src/frontend/context/AppContext.tsx

```diff
@@ -57,6 +56,26 @@ interface AuthProviderExports {
     innloggetSaksbehandler: ISaksbehandler | undefined;
 }
 
+const tilgangModal = (data: IRestTilgang, lukkModal: () => void) => ({
```

Comment:
ekstrahert fra "sjekkTilgang" for å gjenbruke i ny "hentPerson"-metode som brukes fra opprettFagsakModal